src/flamenco/runtime/program/fd_stake_program.c

```diff
@@ -2083,6 +2083,321 @@ redelegate( fd_exec_instr_ctx_t const * ctx,
   return 0;
 }
 
```

Comment:
fix code style

Reply:
not going to nit the entire, file but just fix throughout

Reply:
this will be part of a different pr when stake is audited. redelegate needs to be removed and permalinks need to be updated